Markdown linting and Langauge fixes (#123)
eficode-academy/kubernetes-katas: intro.md

- The image used for the container
- The port the container listens on

> Do not worry if you don't understand everything yet; we will go through it in detail later in the course.

Reviewer: ```suggestion
Do not worry if you don't understand everything yet. We will go through it in detail later in the course.
```

frontend    NodePort   10.96.223.218   <none>        80:32458/TCP   12s
```

> In this example, Kubernetes has chosen port `32458`; you will most likely get a different number.

Reviewer: ```suggestion
In this example, Kubernetes has chosen port `32458`. You will most likely get a different number.
```

Finally, look up the IP address of a node in the cluster with:

``` bash
kubectl get nodes -o wide
```

> > :bulb: The `-o wide` flag makes the output more verbose, i.e., to include the IPs

Reviewer: ```suggestion
> :bulb: The `-o wide` flag makes the output more verbose, i.e., to include the IP addresses.
```
